### DISCUSSION TITLE
Haproxy dashboard in Grafana for OKD/OpenShift

### DISCUSSION THREAD

**Discussion by danielchristianschroeter** [okd-project/okd#778] [Ideas]
It seems there is no default dashboard to monitor the Haproxy/Routes from OKD/OpenShift. It would be great if there would be a default Dashboard included in default cluster monitoring Grafana. Here are some impressions:
https://github.com/rfrail3/grafana-dashboards/blob/master/prometheus/haproxy-2-full.json 
https://github.com/jimmidyson/prometheus-grafana-dashboards
https://grafana.com/grafana/dashboards/2428

**Comment by vrutkovs** [comment]
Including these dashboards in stock Grafana means we'd have to support them for the lifetime of the product. This means we'd have to verify these are valid and sufficient, which is not trivial as it cannot be done via automated tests. So I'm not very optimistic about having the dashboards included.

Another issue is extensibility - default Grafana we ship cannot be modified, which would eventually become important for end users. Perhaps a better way to add more complex dashboards is via Grafana Operator from OperatorHub - as it gives more control to the admins?

Additional monitoring is however always useful - could you create a short guide about these so that OKD users would be aware that these exist and may be useful

**Comment by abdennour** [comment]
since OKD 4.15 , it becomes embdeed in the IDP (called OCP console).
Reverse engineer PQL queries from that
